sphinx/templates/latex/latex.tex_t

```diff
@@ -29,6 +29,7 @@
 <%= fontpkg %>
 <%= fncychap %>
 \usepackage<%= sphinxpkgoptions %>{sphinx}
+\usepackage{sphinxmessage}
```

Comment:
As it would be cumbersome to add a `\ProvidesPackage` to that file, it is simpler to use `\input{sphinxmessage.sty}` or `\makeatletter\input{sphinxmessage.sty}\makeatother` rather if the file contains `@` letter in macros.